router/usecase/router_usecase.go

```diff
@@ -238,6 +232,11 @@ func filterDuplicatePoolIDRoutes(rankedRoutes []route.RouteImpl) []route.RouteIm
 		existsPoolID := false
 
 		for _, pool := range pools {
+			// Skip transmuter pools since they offer no slippage benefits.
+			if pool.GetSQSType() == domain.AlloyedTransmuter || pool.GetSQSType() == domain.TransmuterV1 {
+				continue
+			}
```

Comment:
Note to reviewer: transmuter and alloyed should not be filtered as duplicates as they offer no slippage swaps.